proposals/wg-image-compatibility.md

```diff
@@ -0,0 +1,104 @@
+# OCI Working Group Proposal: Image Compatibility
+
+Proposal created from [OCI WG template](https://github.com/opencontainers/tob/blob/master/WG-TEMPLATE.md).
+
+## Image Compatibility OCI Working Group - Governance Charter
+
+This document describes the basic governance principles for the Image Compatibility Working Group (the “WG”).
+
+The WG operates as an OCI Working Group under the [Open Container Initiative (OCI) Charter](https://github.com/opencontainers/tob/blob/master/CHARTER.md), which describes the responsibilities of the OCI Technical Oversight Board (the "TOB”).
+The WG is established by the TOB as an OCI Working Group pursuant to the OCI Charter.
+Accordingly, the WG will operate in accordance with the OCI Charter and OCI's other policies and procedures, supplemented by the details below.
+
+## Purpose
+
+In mission-critical industries, applications often require special features provided by host operating systems.
+These are usually described in the release notes or installation guides.
+We believe we have identified the missing puzzle in the OCI standard, the image compatibility specification, which is a key feature that describes the requirements of special containerized applications that have hard requirements for specific kernel versions, configurations, modules, or out-of-tree drivers.
+
+For example, a container application free5GC UPF requiring the gtp5g kernel module will only be compatible with kernel 5.0.0-23-generic or 5.4.x due to the module's hard kernel version requirements - https://free5gc.org/guide/3-install-free5gc/.
+If SR-IOV or GPU is required by the container, other requirements have to be specified against the host OS.
+Kata Containers use cases are the best example: https://github.com/kata-containers/kata-containers/tree/main/docs/use-cases.
+
+
+The described incompatibility problem affects bare-metal (e.g. high performance computing) and distributed systems use cases.
```

Comment:
not only bare-metal and istributed systems, even a containerlized application running in virtual machine, the incompatibility may occur between conatiner and guest OS

Reply:
Fixed.